src/sarif_converter.ts

```diff
@@ -69,10 +65,11 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
         level: severity,
       },
       help: {
-        text: suggestion.message,
+        markdown: suggestion.text,
+        text: '',
       },
       properties: {
```

Comment:
is it possible to use property as a property bag and include a cwe and a language key instead of have them in tags? https://docs.oasis-open.org/sarif/sarif/v2.0/csprd02/sarif-v2.0-csprd02.html#_Toc10127639

Reply:
done